src/ritika/exams/Coding_Exam_4_Pg1.java

```diff
@@ -0,0 +1,34 @@
+package ritika.exams;
+
+/*Program 1: WAP to Find Common Element Between Two Arrays
+
+input:
+int array1 [] = [1, 2, 5, 5, 8, 9, 7, 10]
+int array2 [] = [1, 0, 6, 15, 6, 4, 7, 0]
+
+output: 1 7
+*/
+public class Coding_Exam_4_Pg1 {
+
+	static void findCommonElementsBetweenArrays(int[] array1, int[] array2) {
+		if (array1.length == array2.length || array1.length < array2.length) {
+			System.out.println("Common elements:");
+			for (int index = 0; index < array1.length; index++) {
+				if (array1[index] == array2[index])
```

Comment:
Hi Ritika,
you need to compare element in two loop, fortunately both array have same common value on same index, but think on diff index,
Pls update accordingly. I know this is not mentioned but this is like counter question on same program.

Reply:
@harshvegada 
Required changes have been done and code is updated. Time taken to resolve issue is approx 15mins.
Thanks for reviewing!